src/index.html

```diff
@@ -11,7 +11,244 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post"
+    >
+      <fieldset>
+
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input 
+          type="text"  
+          autocomplete="off"
+          minlength="3"
+          maxlength="15"
+          required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input 
+          type="text"  
+          autocomplete="off"
+          minlength="3"
+          maxlength="15"
+          required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input 
+          type="number"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input 
+          type="date"
+          required
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+          class="bottomClass"
+          type="checkbox"
+          required
+          > 
+        </label>
+
+      </fieldset>
```

Comment:
```suggestion
        </label>
      </fieldset>
```
remove empty lines between parent and child elements